src/utils.c

```diff
@@ -141,7 +141,7 @@ char **string_to_array(const char *string, const char *delimiter)
 {
         char **arr = NULL;
         if (string) {
-                arr = g_strsplit(string, ",", 0);
+                arr = g_strsplit(string, delimiter, 0);
```

Comment:
Does it cause a conflict? Because I cannot do without, since `XDG_CONFIG_DIRS` gets split on `:`. Or should I pull https://github.com/dunst-project/dunst/commit/9f15d147b0089939ea4efbfef4a92e3a7bdcc23e?

Reply:
On second though I think it will fine if you don't do anything. This change will probably disappear from the commit when merging.